src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioDocumentNavigationService.cs

```diff
@@ -176,14 +176,16 @@ public bool TryNavigateToPosition(Workspace workspace, DocumentId documentId, in
             return NavigateTo(textBuffer, vsTextSpan);
         }
 
-        private static Document OpenDocument(Workspace workspace, DocumentId documentId, bool usePreviewTab)
+        private static Document OpenDocument(Workspace workspace, DocumentId documentId, OptionSet options)
```

Comment:
if null is a valid value here is there a reason we don't make it an optional parameter?